Functions/Quantification/xASL_quant_FEAST.m

```diff
@@ -34,8 +34,8 @@ function xASL_quant_FEAST(x)
 %% 1. Provide feedback
 fprintf('%s\n', 'Running FEAST quantification, assuming single compartment CBF quantification');
 
-if x.modules.asl.bUseBasilQuantification
-    warning('BASIL quantification detected, FEAST ATT computation might be inaccurate');
+if x.modules.asl.bUseExternalQuantification
+    warning('External quantification detected, FEAST ATT computation might be inaccurate');
```

Comment:
Replace by:

warning('External quantification detected, FEAST ATT computation is not tested yet with external quantification');

Reply:
Done in 160d2997